transport/notmodified/notmodified.go

```diff
@@ -15,6 +16,27 @@ func Response(req *http.Request, header http.Header) *http.Response {
 			return response(req, header)
 		}
 	}
+	if config.LastModifiedEnabled {
+		lastModifiedRaw := header.Get("Last-Modified")
+		if len(lastModifiedRaw) == 0 {
+			return nil
+		}
+		lastModified, err := time.Parse(http.TimeFormat, lastModifiedRaw)
+		if err != nil {
+			return nil
+		}
+		ifModifiedSinceRaw := req.Header.Get("If-Modified-Since")
+		if len(ifModifiedSinceRaw) == 0 {
+			return nil
+		}
```

Comment:
Let's move this up so we check the `If-Modified-Since` existence before parsing `Last-Modified`

Reply:
nice catch. Fixed!